CMakeLists.txt

```diff
@@ -637,7 +637,7 @@ endif()
 if(NGRAPH_CPU_ENABLE OR NGRAPH_GENERIC_CPU_ENABLE)
     include(cmake/external_eigen.cmake)
 endif()
-if(NGRAPH_CPU_ENABLE)
+if(NGRAPH_CPU_ENABLE OR NGRAPH_MLIR_ENABLE)
```

Comment:
Is this actually necessary?

Reply:
It is. MLIR is now separate from CPU and MLIR uses oneDNN callbacks.

Reply:
I forgot about oneDNN callbacks.  Thanks.